index.html

```diff
@@ -0,0 +1,49 @@
+    <meta name="viewport" content="width=assets, initial-scale=1.0">
+    <title>Unspalsh</title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+
+<body>
+    <main>
+        <div class="flex justify-evenly">
+            <div class="flex column gallery-column">
+                <img src="./assets/anita-austvika-T5Y8iKuhr_U-unsplash.jpg
+            " alt="img1" loading="lazy" class="my-2">
+                <img src="./assets/anita-austvika-dROLarwZ7p8-unsplash.jpg" alt="img2" loading="lazy" class="my-2">
+                <img src="./assets/anita-austvika-f2imCQ-wfso-unsplash.jpg" alt="img3" loading="lazy" class="my-2">
+                <img src="./assets/alec-favale-Bi_5VsaOLnI-unsplash.jpg" alt="img4" loading="lazy" class="my-2">
+                <img src="./assets/anita-austvika-tMRr6G2dH70-unsplash.jpg" alt="img5" loading="lazy" class="my-2">
+                <img src=" ./assets/austin-poon-JO_S6ewBqAk-unsplash.jpg" alt="img6" loading="lazy" class="my-2">
+                <img src="./assets/charmore-nel-DoN9xerfCe8-unsplash.jpg" alt="img7" loading="lazy">
+                <img src="./assets/dmitriy-k-pF2NIUme_Y0-unsplash.jpg" alt="img8" loading="lazy" class="my-2">
+            </div>
+            <div class="flex column gallery-column">
+                <img src="./assets/element5-digital-ceWgSMd8rvQ-unsplash.jpg" alt="img9" loading="lazy" class="my-2">
+                <img src="./assets/eugenivy_now-uRR-RToN4HY-unsplash.jpg" alt="img9" loading="lazy" class="my-2">
+                <img src="./assets/georgi-kalaydzhiev-AG8AGJA8_e4-unsplash.jpg" alt="img10" loading="lazy" class="my-2">
+                <img src="./assets/jazmin-quaynor-FoeIOgztCXo-unsplash.jpg" alt="img10" loading="lazy" class="my-2">
+                <img src="./assets/mailchimp-0qnRfgnZIsI-unsplash.jpg" alt="img9" loading="lazy" class="my-2">
+                <img src="./assets/mathilde-langevin-uKJ17IIgYZI-unsplash.jpg" alt="img10" loading="lazy" class="my-2">
+                <img src="./assets/mathilde-langevin-W3tBP-L8T60-unsplash.jpg" alt="img10" loading="lazy" class="my-2">
+                <img src="./assets/mk-s-J06P-vc1JKk-unsplash.jpg" alt="img9" loading="lazy" class="my-2">
+            </div>
+            <div class="flex column gallery-column">
+                <img src="./assets/philipp-pilz-JsOlctJwniY-unsplash.jpg" alt="img10" loading="lazy" class="my-2">
+                <img src="./assets/photo-1667713551609-bf9b81db0095.jpg" alt="img10" loading="lazy" class="my-2">
+                <img src="./assets/rodion-kutsaiev-r_NDpTNtgW0-unsplash.jpg" alt="img9" loading="lazy" class="my-2">
+                <img src="./assets/ronny-rondon-mMlpDj0yZn4-unsplash.jpg" alt="img10" loading="lazy" class="my-2">
+                <img src="./assets/tanya-shulga-Vc0GoIzva-Q-unsplash.jpg" alt="img10" loading="lazy" class="my-2">
+                <img src="./assets/timo-wagner-fT6-YkB0nfg-unsplash.jpg" alt="img9" loading="lazy" class="my-2">
```

Comment:
In situations like this when you have a class applying to multiple elements, it's better to have it done via CSS.